### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/crowdin-cli/commit/672752d85fb11c43da282dc17019c99c7b8ffb26#commitcomment-88832249

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/crowdin-cli/commit/bded2e06d6dccabe808303e9a1c6d98e2afce189

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -51,7 +51,7 @@
 #       and GRADLE_OPTS) rely on word-splitting, this is performed explicitly;
 #       see the in-line comments for details.
 #
-#       There are tweaks for specific operating systems such as AIX, CygWin,
+#       There are tweaks for specific operating systems such as AIX, Cygwin,
```
Comment (jsoref) [inline gradlew]: Brand

File: src/main/resources/messages/messages.properties [hunk 2]
```diff
@@ -168,7 +168,7 @@ crowdin.string.delete.usage.description=Delete source string
 crowdin.string.delete.usage.customSynopsis=@|fg(green) crowdin string delete|@ [CONFIG OPTIONS] [OPTIONS]
 crowdin.string.delete.id=Numeric string identifier (multiple ids could be specified)
 crowdin.string.delete.text=Use text as string identifier (several texts can be specified)
-crowdin.string.delete.identifier=String identifier identifier (several identifiers can be specified)
+crowdin.string.delete.identifier=String identifier (several identifiers can be specified)
```
Comment (jsoref) [inline src/main/resources/messages/messages.properties:171]: ??

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

File: gradlew (hunk 1 above)
Comment (andrii-bodnar) [reply gradlew]: yes, let's not change this

File: src/main/resources/messages/messages.properties (hunk 2 above)
Comment (andrii-bodnar) [reply src/main/resources/messages/messages.properties:171]: The updated version is correct

**Comment by andrii-bodnar** [comment]
@jsoref thanks a lot for your contribution! 🚀 

The minor thing that needs to be done is to revert the changes in the `gradlew` file, everything else looks good to me!

File: gradlew (hunk 1 above)
Comment (jsoref) [reply gradlew]: By brand, I mean that https://www.cygwin.com/ writes their brand `Cygwin` not `CygWin`. That said. Dropped.

**Comment by andrii-bodnar** [comment]
@jsoref thank you!

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**